### DISCUSSION TITLE
Changes on upcoming v2.1.0.0

### DISCUSSION THREAD

**Discussion by K-vanc** [K-vanc/Tempest-EPG-Generator#157] [Announcements]
Dear Tempest Users;

I am planning to release new version 2.1.0.0 soon. This new version has several bug fixes and improvements but main change will be usage of new faster and more secure encryption module. With upgrade of this module, existing [ENC] and [ENC][EX] siteconfigs will continue to work as normal on 2.1.0.0 as part of backward compatibility but newly encrypted(or modified) siteconfigs after release of 2.1.0.0 will only work with Tempest versions >= 2.1.0.0. 

Currently, I don't have any plan to re-update existing encrypted siteconfigs unless we face with an unexpected compatibility error.

This is an advance notice for keeping your Tempest updated

**Comment by donwolfoz-lab** [comment]
hello, im look in the manual for save merged xml to custom location like this

tempest.php --xmlmerge ml=file1 xml=file2  out=/some/new/location/final gz

**Comment by K-vanc** [comment]
hi,

xmlmerger output will be always /epg subfolder. You cannot change it via tempest command because php has no working permit out of root folder(with default settings). So you have to use your system commands to copy/move created file to another directory. If you are using Unix, you can pipe or add another line into .sh file for copy;

`tempest.php --xmlmerge xml=file1 xml=file2 out=final gz`
`cp /your/root/folder/tempest_config/epg/final.xml.gz /path/of/your/folder/`

you can do the same on Windows with a .bat file and "copy" command;

`tempest.php --xmlmerge xml=file1 xml=file2 out=final gz`
`copy C:\Your\root\folder\tempest_config\epg\final.xml.gz C:\path\of\your\folder\final.xml.gz`

Edit: fix your merge command since first file is "ml" parameter instead of "xml"
